js/helpers/ng-tracks.js

```diff
@@ -0,0 +1,188 @@
+                if(startCountingAt.match(/^(after|before)/)) {
+                    console.error(`Illegal trigger: ${str}`);
+                    return undefined;
+                }
+                return {
+                    event: startCountingAt,
+                    timeAfter: direction * quantity
+                };
+            }
+
+        // The mm:ss case
+        } else if(match = str.match(/^(\d+):(\d+)$/)) {
+            let minutes = parseInt(match[1]);
+            let seconds = parseInt(match[2]);
+            let time = minutes * 60 + seconds
+            return time.toString();
+        // The generic word case
+        } else {
+            //Remove all ignorable trigger words
+            str = str.replace(/^(on |with )/, '');
+            return str;
+        }
+    }
+
+    function resolveListeners(trackConfig, track, generalConfig) {
+        var listeners = [];
+
+        // Iterates over all possible actions,
+        // If and action has an attribute for it, it will assign it accordingly
+        Object.keys(ACTIONS).forEach(function(action) {
+            if(!trackConfig.hasOwnProperty(action)) return;
+
+            let event = resolveTimeEvent(trackConfig[action], generalConfig);
+            if(typeof event === 'string') {
+                listeners.push({
+                    name: trackConfig.name,
+                    event: event,
+                    track: track,
+                    action: action
+                });
+            // This is in case of complecated events, i.e. start: 50s after start
+            // In that case, the resolveTimeEvent will return an object,
+            // with a property "event" containing the event to start counting from (in our case "start")
+            // and a property "timeAfter" containing the time to set the event to be triggered after the inital event.
+            // For this there is a special action 'set' which will set a new event when the initial event triggers
+            // Because that's the firsdt time we can know the time of the new event.
+            } else if(typeof event !== undefined) {
+                listeners.push({
+                    name: trackConfig.name,
+                    event: event.event,
+                    timeAfterEvent: event.timeAfter,
+                    track: track,
+                    action: 'set',
+                    metaAction: action
+                });
+            }
+
+            if(trackConfig.name) {
+                track.onEnded(function() {
+                    // Notice: this is the only place we don't pass the time parameter,
+                    // Because there can't be a trigger "302 after after start",
+                    // That's illegal.
+                    trigger(`after ${trackConfig.name}`);
+                });
+            }
+        });
+        return listeners;
+    }
+
+    return {
+        init: function() {
+            return $config.init().then(function(config) {

```

Comment:
$config.init has been removed, this does not work

Reply:
Fixed it